.github/workflows/go.yml

```diff
@@ -0,0 +1,25 @@
+name: Go
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+
+    - name: Build
+      run: go build -v ./...
+
+    - name: Test
+      run: go test -v ./...
```

Comment:
Do you think instead of having to comment and uncomment `t.Skip` every time, this might be better?

```
go test `go list ./... | grep -v ./integration-tests`
```

Reply:
Comitted that suggestion. Looks like it does the trick and is cleaner.